docs/content/getting-started/installation/troubleshooting-installation.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshooting Installation | Dagster
+description: Resources for getting help and troubleshooting common Dagster installation problems, ex. Python and its dependencies
+---
+
+# Troubleshooting Installation
+
+This page covers common troubleshooting issues when installing Dagster. You will also you find resources for getting more help.
+
+---
+
+## Resources
```

Comment:
🤔 Thoughts on making this the last section on this page? I wonder if users wouldn't jump right to these resources instead of self-serving (specifically, seeing if their issue is in this guide), which is what we want them to do.

Reply:
True! I intentionally moved it up because I was conservative in thinking that our troubleshooting page wouldn't cover enough, but you're right in that we _want_ them to find their solution on the page and the resources are the last effort.